source/preface_third.Rmd

```diff
@@ -25,111 +25,87 @@ The book in your hands, or on your screen, is the third edition of a book
+in teaching [@simon1969basic]. He then saw that it was much more efficient to
+build these models with simple computer code, and the result was the first and
+second editions of this book, with their associated software, the *Resampling
+Stats* language — of which more later.
+
+Simon's second conclusion follows from the first. Now that Simon had stripped
+away the unnecessary barrier of mathematics, he had got to the heart of what
+is interesting and difficult in statistics.  Drawing conclusions from noisy
+data involves a lot of hard, clear thinking.  We need to be honest with our
+students about that; statistics is hard, not because it is obscure (it need
+not be), but because it deals with difficult problems. It is exactly that hard logical thinking that can make statistics so
+interesting to our best students; "statistics" is just reasoning about the
+world when the world is noisy.  Simon writes eloquently about this in a
+section in the introduction — "Why is statistics such a difficult subject"
+(\@ref(stats-difficult)).
+
+We needed both of Simon's conclusions to get anywhere.   We cannot hope to
+teach two hard subjects at the same time; mathematics, and statistical
+reasoning. That is what Simon has done: he replaced the mathematics with
+something that is much easier to reason about. Then he can concentrate on the
 real, interesting problem — the hard thinking about data, and the world it
-comes from. To quote from a later section in this book (\@ref(resamp-differs)):
-"Once we get rid of the formulas and tables, we can see that statistics is a
-matter of *clear thinking, not fancy mathematics*."  We hope you will share our
-own experience of thinking this way — that it is tremendously satisfying.
-Instead of thinking "where would I look up the right recipe for this", you find
-yourself asking "what kind of world do these data come from?" and "how can I
-reason about that world?".  We know we cannot persuade you of this in few lines
-in a preface, but we can at least we offer you our own experience, of how
-liberating and refreshing this can be, for teachers and for students.
-
-## Resampling and data science
-
-The ideas in Simon's book, first published in 1992, have found themselves in
-the center of the modern movement of *data science*.  We say more about what we
-mean by *data science* below, but for now, let us say it is some combination of
-statistics and computing.
+comes from. To quote from a later section in this book
+(\@ref(resamp-differs)): "Once we get rid of the formulas and tables, we can
+see that statistics is a matter of *clear thinking, not fancy mathematics*."
+Instead of asking "where would I look up the right recipe for this", you
+find yourself asking "what kind of world do these data come from?" and "how
+can I reason about that world?".  Like Simon, we have found that this way of
+thinking and teaching is almost magically liberating and satisfying.  We hope
```

Comment:
I would avoid "almost magically" throughout the book.  It is not magical at all!  It is very clear why.